From 2729e17a282188cf200530a12a78f9051d55cfc7 Mon Sep 17 00:00:00 2001
From: Ofir Fedida <ofedida@marvell.com>
Date: Wed, 13 Jan 2016 22:58:10 +0200
Subject: [PATCH 0689/1240] a3700: config: add support for DDR code

- added CONFIG_MVEBU_MK6_SUPPORT - support for DDR driver

Change-Id: I91eb4303409552e8d5bb785862cf817bd5589ee9
Signed-off-by: Ofir Fedida <ofedida@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/26792
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hua Jing <jinghua@marvell.com>
---
 configs/mvebu_armadalp_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/mvebu_armadalp_defconfig b/configs/mvebu_armadalp_defconfig
index a41cbb6..7b65db2 100644
--- a/configs/mvebu_armadalp_defconfig
+++ b/configs/mvebu_armadalp_defconfig
@@ -39,4 +39,4 @@ CONFIG_CMD_MVEBU_BUBT=y
 +S:CONFIG_OF_EMBED=y
 +S:CONFIG_DEFAULT_DEVICE_TREE="armada-lp-db"
 +S:CONFIG_MVEBU_COMPHY_SUPPORT=y
-+S:# CONFIG_MVEBU_SPL_DDR_SUPPORT is not set
++S:CONFIG_MVEBU_MK6_SUPPORT=y
-- 
1.9.1

